packages/communication/src/wallet.ts

```diff
@@ -0,0 +1,26 @@
+import { EIP712TypedData } from '@celo/utils/lib/sign-typed-data-utils'
+import { Address, CeloTx, EncodedTransaction, RLPEncodedTx } from './commons'
+
+export interface ReadOnlyWallet {
+  getAccounts: () => Address[]
```

Comment:
Maybe it's too late or out of scope but should a Wallet be 1:1 with an account like Ethers.Js does:
https://docs.ethers.io/v5/api/signer/#Wallet

It would simplify this interface, we wouldn't need callers to pass in their own address every time. But it may create complexity in other places, just thinking out loud.